static-site/pages/index.jsx

```diff
@@ -1,3 +1,8 @@
+/**
+ * 
+ * Example of  dynamic import but using ssr (which is the default dynamic import!)
```

Comment:
It took me a while to work out the difference between a dynamic import as used here and a typical import as used in the /team page. TL;DR we should avoid dynamic imports and just use normal imports.

In terms of build-time rendered / server-rendered HTML there is no difference.

The difference comes client-side, where the [next.js docs explain](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading):

> By using next/dynamic, the [dynamically imported] component will not be included in the page's initial JavaScript bundle. The page will render the Suspense fallback first, followed by the [dynamically imported] component when the Suspense boundary is resolved.

So when we use a `<Link>` to perform a client-side navigation to the splash page we will initially get the fallback which is then replaced by the contents. So it's better to avoid these entirely and just use normal imports. If you want to try this out it helps to add in a `loading: () => (<p>Loading...</p>)` option to the dynamic import. (The reason I used dynamic imports in the first place is to force the site into CSR mode.)

It took me a while to understand this because I thought next.js was using the approach described in [New Suspense SSR Architecture in React 18](https://github.com/reactwg/react-18/discussions/37) where suspended components will / can be replaced by their fallback component in the streamed HTML, and then a later part of the HTML will include the contents of the component + a small script to tell the browser to replace the fallback. But that's not in play here.